docs/src/main/docs/webserver/08_json-support.adoc

```diff
@@ -56,16 +56,19 @@ Routing.builder()
 [source,java]
 .Handler that receives and returns JSON objects
 ----
-private void sayHello(ServerRequest req, ServerResponse res, JsonObject json) { // <1>
-        JsonObject msg = Json.createObjectBuilder()   // <2>
+private static final JsonBuilderFactory jsonFactory = Json.createBuilderFactory(null); <1>
```

Comment:
I would rather use an empty map than null.

Reply:
Looking at the docs and the implementation, passing `null` or an empty map results in the exact same behavior.

Reply:
Yes, `null` is documented as valid, but I've gone ahead and changed these to `Collections.emptyMap()` since it's a bit more self-documenting.

The JSON-P bug is https://github.com/eclipse-ee4j/jsonp/issues/154